### PR TITLE
Add a function to prepare body for trigger topic message

### PR DESCRIPTION
**Pull request by tapishr**
## Summary by CodeRabbit

- **Refactor**
	- Improved data preparation process for triggering actions in the extension API.
	- Cleaned up the `parseURL` function by removing redundant lines.

### CODE REVIEW

**Review by MuskanPaliwal — APPROVED** [review]

File: pages/api/extension/trigger.ts [hunk 1]
```diff
@@ -105,4 +105,12 @@ function parseURL(url: string) {
...
-
+function prepareBody(repoProvider: string, repoOwner: string, repoName: string, prNumber: string, repoConfig: any) {
+    return {
+        repo_provider: repoProvider,
+        repo_owner: repoOwner,
+        repo_name: repoName,
+        pr_number: prNumber,
+        repo_config: repoConfig,
+    }
+}
```
Comment (avikalpg) [inline pages/api/extension/trigger.ts:116]: Any reason why you decided to turn this single line assignment into a function call?

**Review by avikalpg — CHANGES_REQUESTED** [review]
Creating a function when there is no added reusability or abstraction can make the code less readable.

**Review by MuskanPaliwal — DISMISSED** [review]
I agree with @avikalpg's comment

File: pages/api/extension/trigger.ts (hunk 1 above)
Comment (tapishr) [reply pages/api/extension/trigger.ts:120]: no reason, fixing

File: pages/api/extension/trigger.ts [hunk 2]
```diff
@@ -65,7 +65,7 @@ async function triggerDPU(url: string, userEmail: string) {
             return {auto_assign: false, comment: false};
         });
     // prepare body
-    
+    const triggerBody = prepareBody(repoProvider, repoOwner, repoName, prNumber, repoConfig);
```
Comment (avikalpg) [inline pages/api/extension/trigger.ts:68]:
```suggestion
    const triggerBody = { repoProvider, repoOwner, repoName, prNumber, repoConfig };
```

**Review by avikalpg — DISMISSED** [review]

**Review by avikalpg — APPROVED** [review]

**Pull request merged**